PacketTunnelProvider/AppTrackingProtectionPacketTunnelProvider.swift

```diff
@@ -29,6 +29,34 @@ class AppTrackingProtectionPacketTunnelProvider: NEPacketTunnelProvider {
     let proxyServerAddress = "127.0.0.1"
     var proxyServer: GCDHTTPProxyServer!
     
+    override init() {
+        super.init()
+        
+        let source = DispatchSource.makeMemoryPressureSource(eventMask: .all, queue: nil)
+
+        let q = DispatchQueue.init(label: "test")
```

Comment:
Do the queue & dispatch source need to be retained somewhere? Won't they get deallocated at the end of the init function's scope?